cloud/google/node_init.sh

```diff
@@ -42,9 +42,9 @@ fi
 sudo bash ./${AGENT_INSTALL_SCRIPT}
 
 # Examine what kind of Docker image is on this machine to determine what to log.
-if docker images | grep ct-server; then
+if docker images | grep '/ct-server$'; then
   CT_LOGS_PREFIX="${DATA_DIR}/ctlog/logs/ct-server"
-elif docker images | grep ct-mirror; then
+elif docker images | grep '/ct-mirror$'; then
```

Comment:
Good point, I've updated this code to to make `docker images` write only the repository names (since that's what we're examining) to a file and then grep that.